website/static/js/keen.js

```diff
@@ -0,0 +1,143 @@
+var keen = require('keen-js');
+var oop = require('js/oop');
+var uuid = require('uuid');
+
+var KeenTracker = oop.defclass({
+    constructor: function(keenProjectId, keenWriteKey) {
+        this.keenClient = new keen({
+            projectId: keenProjectId,
+            writeKey: keenWriteKey
+        });
+        this.init();
+    },
+
+    createOrUpdateKeenSession: function() {
+        var date = new Date();
+        var min = 25;
```

Comment:
Minor: consider giving this a better name, like `expiresInMinutes`.  `min` is quite generic.